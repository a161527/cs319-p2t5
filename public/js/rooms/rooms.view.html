<!-- <div ui-view> -->
	<div ng-show='noUsers'>No users currently requiring accomodations</div>
	<div ng-hide='noUsers'>
		<h4>Select Attendees</h4>
<!-- 		<label class='checkbox-inline'>
			<input type="checkbox" 
				ng-model='dependentsSelectAll'
				ng-click='selectAll("selected", dependentsSelectAll, dependents)'>
			Select All
		</label> -->

		<div class="list-group">
			<button type="button" 
				class="list-group-item" 
				ng-repeat='(idx, value) in dependents' 
				ng-click='dependents[idx].selected = !dependents[idx].selected'>
				<div class='row'>
					<div class='col-md-2'>
						<input type="checkbox" ng-model='dependents[idx].selected'>
					</div>
					<div class='col-md-2'>
						<span class='glyphicon glyphicon-user glyphicon-large'></span>
					</div>
					<div class='col-md-8'>
						<div>{{dependents[idx].user.firstName}} {{dependents[idx].user.lastName}}</div>
					</div>
				</div>
			</button>
		</div>
		<error-message ng-show='noSelection'>There must be one user selected<span class="glyphicon glyphicon-remove pull-right hand-cursor" ng-click='removeMessage()'></span></error-message>
		<br>
		<div class='row'>
			<div class='col-md-6'>
				<button class='btn btn-default btn-block' ng-click='cancel()'>Cancel</button>
			</div>
			<div class='col-md-6'>
				<button class='btn btn-primary btn-block' ng-click='next()'>Next</button>
			</div>
		</div>
	</div>
<!-- </div> -->