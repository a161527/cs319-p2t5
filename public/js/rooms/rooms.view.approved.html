<div class='row'>
	<div class='col-sm-9'>
		<h4>Room Assignments</h4>
	</div>
	<div class='col-sm-3'>
		<button class='btn btn-info btn-block' 
			ng-click='back()'>
			Go to Assign Rooms
		</button>
	</div>
</div>
<div ng-show='roomsAssigned.length === 0'>No rooms assigned</div>
<ul class="list-group">
	<div ng-repeat='room in roomsAssigned'>
		<li class='list-group-item unselectable-text'>
			<div class='row'>

				<div class='col-sm-3'>
					<span class='glyphicon glyphicon-bed glyphicon-large'></span>
				</div>
				<div class='col-sm-6'>

					<div class='row'>
						<div class='col-sm-6'>
							<b>Name:</b>
						</div>
						<div class='col-sm-6'>
							{{room.registration.user.firstName}} {{room.registration.user.lastName}}
						</div>
					</div>
					<div class='row'>
						<div class='col-sm-6'>
							<b>Room Name:</b>
						</div>
						<div class='col-sm-6'>
							{{room.roomName}}
						</div>
					</div>
					
				</div>

			</div>
		</li>
	</div>
</ul>